packages/components/src/styles/components/application-state.scss

```
@@ -7,43 +7,81 @@
 // APPLICATION-STATE COMPONENT
 //
 
-$hds-application-state-padding: 12px 0;
+$hds-application-state-content-max-width: 480px;
 
 .hds-application-state {
-  width: 19.5rem;
-  max-width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  width: fit-content;
   margin: 0 auto; // this will center the component in the parent container
+
+  &.hds-application-state--align-center {
+    align-items: center;
+    text-align: center;
+
+    .hds-application-state__header,
+    .hds-application-state__body,
+    .hds-application-state__footer {
+      width: auto;
+    }
+  }
+}
+
+.hds-application-state__media {
+  margin-bottom: 32px;
+
+  > * {
+    display: block;
+    max-width: 100%;
+  }
 }
 
 .hds-application-state__header {
   display: grid;
+  grid-template-rows: auto auto;
   grid-template-columns: min-content 1fr;
-  align-items: start;
-  color: var(--token-color-foreground-faint);
+  align-items: center;
+  width: 100%;
+  max-width: $hds-application-state-content-max-width;
+  color: var(--token-color-foreground-strong);
+}
+
+.hds-application-state__error-code {
+  grid-column: 1 / -1;
 }
```

There doesn't seem to be a 4px gap between the `errorCode` and `Icon` + `Title`
```suggestion
    margin-bottom:4px;
}
```

---

> Also is this the common convention if we wanted the text to always be uppercase?

We don't know if consumers may want to use it for other "messages" so let's keep it agnostic. It's easy for them to make it uppercase if they want.

---

Ah I see what you're saying, that's fair.